api/src/main/resources/workbench-api.yaml

```diff
@@ -235,6 +235,37 @@ paths:
       responses:
         204:
           description: Request Received.
+
+  "/v1/validateTermsOfServiceStatus":
+    get:
+      tags:
+        - profile
+      description: Validates if the logged in user has accepted the latest version of All of Us and Terra service
+      operationId: validateUserTermsOfServiceStatus
+      responses:
+        204:
+          description: Logged in User has accepted latest Version of All of Us and Terra TOS
+        400:
```

Comment:
we should reserve error codes for exceptional states.  It will be common for this endpoint to return false here, so we should return 200/"false" or similar